### DISCUSSION TITLE
"dual watch after" timings..

### DISCUSSION THREAD

**Discussion by nikant** [egzumer/uv-k5-firmware-custom#526] [Q&A] [answered]
hi all,

to make DW more responsive does anyone know if there will be a problem lowering these values?

> const uint16_t    dual_watch_count_after_tx_10ms   =  3600 / 10;   // 3.6 sec after TX ends
> const uint16_t    dual_watch_count_after_rx_10ms   =  1000 / 10;   // 1 sec after RX ends ?
> const uint16_t    dual_watch_count_after_1_10ms    =  5000 / 10;   // 5 sec
> const uint16_t    dual_watch_count_after_2_10ms    =  3600 / 10;   // 3.6 sec

Thank you all for your time! I have learned a lot

**Comment by egzumer** [comment, marked as answer]
I don't think anyone will answer that. You have to experiment and see if that is what you want.
Default delays prioritize listening to last TX or RX frequency to keep reliable communication on the one that is currently used, and not to catch each and every transmission that might occur on both frequencies at any given time. Quansheng has only one receiver, dual watch is just a substitute and certain trade-offs has to be made.

**Comment by nikant** [comment]
Thank you, I was afraid that those would mess another process but I couldn't find anything specific in the code..
Trial and error it is then!
